### ISSUE TITLE
Training setting for the tiny version

### ISSUE THREAD

**Issue by jingweizhang-xyz** [wangf3014/Adventurer#3]
Hi,

Thank you for your great work! I currently working on a structure and want to apply my structure to your method. When I was reproducing the your tiny version. I followed the hyper-parameters in the Supplementary Material that only using the stage2 recipe of the small version for 300 epochs. But I got a final max top-1 accuracy of 77.8% for the non-ema version and 77.6% for the ema version, which is a bit lower than the reported 78.2%. I assume it should be a hyper-parameter issue. Could you please help me find out what's wrong with my training? 

Here is my training command:
torchrun --standalone --nnodes=1 --nproc-per-node=8 main.py --model adventurer_tiny_patch16  --data-path xxx --batch 128 --lr 5e-4 --weight-decay 0.05 --output_dir xxxx --reprob 0.0 --smoothing 0.0  --repeated-aug --ThreeAugment --epochs 300 --input-size 224 --drop-path 0.2 --dist-eval 

I can share my training log if you need it.

Thank you very much!

**Comment by wangf3014** [comment]
Did you try --drop-path 0.1? That would be the problem.

**Comment by jingweizhang-xyz** [comment]
Got it. Let me have a try.

**Comment by jingweizhang-xyz** [comment]
It works,  thank you!
